SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -394,12 +394,14 @@ void UGlobalStateManager::HandleActorBasedOnLoadBalancer(AActor* Actor) const
 	// Replicated level Actors should only be initially authority if:
 	//  - these are workers starting as part of a fresh deployment (tracked by the bCanSpawnWithAuthority bool),
 	//  - the load balancing strategy says this server should be authoritative (as opposed to some other server).
-	bAuthoritative |= bCanSpawnWithAuthority && NetDriver->LoadBalanceStrategy->ShouldHaveAuthority(*Actor);
+	bAuthoritative |= (bCanSpawnWithAuthority || Actor->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_NotPersistent))
```

Comment:
What's  the purpose of this change?

Reply:
This has to do with actors that are in the level and marked with NotPersistent ending up without having anybody with authority over them. This solution has been discussed with @m-samiec.

Reply:
Adding something, let me know if it's good enough